lib/ace/ext/papeeria/tex_completer.coffee

```diff
@@ -188,6 +167,29 @@ define((require, exports, module) ->
         @refCache = new CompletionsCache(processReferenceJson)
         @citeCache = new CompletionsCache(processCitationJson)
       @init: (editor) ->
+        init = (editor, bindKey) ->
```

Comment:
come on. What's the point of having function here? Why don't you just execute its body directly?